### PR TITLE
Response proto

### PR DESCRIPTION
**Pull request by Torbacka**
Added response proto and repeating field

### CODE REVIEW

File: src/main/proto/Payload.proto [hunk 1]
```diff
@@ -10,6 +10,7 @@ message Payload {
     string queryString = 5;
     string body = 6;
     fixed64 flakeId = 7;
+    bool repeating = 8;
```
Comment (mantono) [inline src/main/proto/Payload.proto:13]: What purpose does this serve?

**Review by mantono — COMMENTED** [review]
Looks good. I imagine that we might have to extend Response in the future (all tough we should not get ahead of ourselves), time will tell.

I feel that I can only give limited feedback since I am still not very familiar with the circumstances during which still will be used.

File: src/main/proto/Payload.proto (hunk 1 above)
Comment (Torbacka) [reply src/main/proto/Payload.proto:13]: If you want the request to be repeating you need to set this bool. Its used in Katie

Comment (williamhogman) [reply src/main/proto/Payload.proto:13]: Not a super-fan of calling this repeating but I have no better ideas.

**Review by williamhogman — APPROVED** [review]
LGTM

**Pull request merged**